### PR TITLE
Vue3 migration

### PR DESCRIPTION
**Pull request by alan-wu**

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -1,9 +1,7 @@
...
-import FlatmapVuer from "./FlatmapVuer.vue";
-import MultiFlatmapVuer from "./MultiFlatmapVuer.vue";
-  
-export {
-  FlatmapVuer,
-  MultiFlatmapVuer
-};
+import FlatmapVuer from './FlatmapVuer.vue'
+import MultiFlatmapVuer from './MultiFlatmapVuer.vue'
+import Tooltip from './Tooltip.vue'
```
Comment (Tehsurfer) [inline src/components/index.js:5]: Does the tooltip get used externally now?

File: package.json [hunk 2]
```diff
@@ -15,42 +14,54 @@
     "url": "https://github.com/ABI-Software/flatmapvuer.git"
   },
   "scripts": {
-    "start": "vue-cli-service serve",
-    "serve": "vue-cli-service serve --port 8082",
+    "serve": "vite --host",
     "build": "vue-cli-service build --dest test_html --mode staging",
-    "build-bundle": "vue-cli-service build --target lib --name flatmapvuer ./src/components/index.js",
+    "build-bundle": "vite build",
     "lint": "vue-cli-service lint",
```
[the remark this answers is not in the document]
Comment (alan-wu) [reply package.json:20]: Remove vue-cli completely please.

File: src/components/FlatmapVuer.vue [hunk 3]
```diff
@@ -69,261 +67,286 @@
...
-            class="icon-button fitWindow"
-            @click.native="resetView()"
-            @mouseover.native="showToolitip(2)"
-            @mouseout.native="hideToolitip(2)"
-          />
+          <template #reference>
+            <map-svg-icon
+              icon="fitWindow"
+              class="icon-button fitWindow"
+              @click.native="resetView()"
```
[the remark this answers is not in the document]
Comment (alan-wu) [reply src/components/FlatmapVuer.vue:196]: https://v3-migration.vuejs.org/breaking-changes/v-on-native-modifier-removed.html

File: src/components/index.js (hunk 1 above)
Comment (alan-wu) [reply src/components/index.js:5]: Yes

File: src/components/FlatmapVuer.vue [hunk 4]
```diff
@@ -69,261 +67,286 @@
...
+            >
+              <el-icon><el-icon-arrow-left /></el-icon>
+            </div>
+          </div>
+        </template>
+      </el-popover>
       <el-popover
+        v-if="openMapRef"
         ref="open-map-popover"
+        :virtual-ref="openMapRef"
```
[the remark this answers is not in the document]
Comment (alan-wu) [reply src/components/FlatmapVuer.vue:343]: https://element-plus.org/en-US/component/popover.html#virtual-triggering

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (alan-wu) [reply package.json:20]: The one not being removed are waiting for replacement.

**Pull request merged**